### PR TITLE
fix: blank line after table

### PR DESCRIPTION
**Pull request by hyoban**

### CODE REVIEW

**Comment by hyoban** [comment]
fix #298

File: web/src/labs/marked/parser/Table.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@
  * | 1 | 2 | 3 |
  * | 4 | 5 | 6 |
  */
-export const TABLE_REG = /^(\|.*\|)(?:(?:\n(?:\|-*)+\|))((?:\n\|.*\|)+)/;
+export const TABLE_REG = /^(\|.*\|)(?:(?:\n(?:\|-*)+\|))((?:\n\|.*\|)+)\n/m;
```
Comment (boojack) [inline web/src/labs/marked/parser/Table.ts:9]:
```suggestion
/^(\|.*\|)(?:(?:\n(?:\|-*)+\|))((?:\n\|.*\|)+)(\n?)/
```
```suggestion
export const TABLE_REG = /^(\|.*\|)(?:(?:\n(?:\|-*)+\|))((?:\n\|.*\|)+)(\n?)/;
```

File: web/src/labs/marked/parser/Table.ts [hunk 2]
```diff
@@ -35,7 +35,8 @@ const renderer = (rawStr: string): string => {
   <tbody>
     ${tableBody.map((row) => `<tr>${row.map((str) => `<td>${str}</td>`).join("")}</tr>`).join("")}
   </tbody>
-</table>`;
+</table>
+`;
```
Comment (boojack) [inline web/src/labs/marked/parser/Table.ts:39]:
```suggestion
</table>${matchResult[3]}`;
```

File: web/src/labs/marked/marked.test.ts [hunk 3]
```diff
@@ -6,11 +6,15 @@ describe("test marked parser", () => {
   test("test markdown table", () => {
     const tests = [
       {
-        markdown: `| a | b | c |
+        markdown: `aaa
```
Comment (boojack) [inline web/src/labs/marked/marked.test.ts:9]: Use a real data will be better.

**Review by boojack — CHANGES_REQUESTED** [review]
We can use the same way as other syntax. e.g. https://github.com/usememos/memos/blob/main/web/src/labs/marked/parser/CodeBlock.ts#L4

File: web/src/labs/marked/parser/Table.ts (hunk 1 above)
Comment (boojack) [inline web/src/labs/marked/parser/Table.ts:9]:
```suggestion
export const TABLE_REG = /^(\|.*\|)(?:(?:\n(?:\|-*)+\|))((?:\n\|.*\|)+)(\n?)/;
```

**Review by boojack — APPROVED** [review]
LGTM

**Pull request merged**